internal/aws/resource_aws_idc_integration.go

```diff
@@ -95,8 +97,16 @@ func (r *AWSIDCIntegrationResource) Schema(ctx context.Context, req resource.Sch
 				MarkdownDescription: "The ARN of the role to assume in order to read AWS IAM Identity Store data",
 				Required:            true,
 			},
+			"audit_role_name": schema.StringAttribute{
+				MarkdownDescription: "The name of the role to assume in each AWS Account in order to read resources",
+				Required:            true,
```

Comment:
This should be optional to prevent a breaking change - we can default to `common-fate-audit` if unset